azure-iot-device/samples/pnp/pnp_environmental_monitor.py

```diff
@@ -0,0 +1,131 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+import asyncio
+import random
+import logging
+
+from azure.iot.device.aio import IoTHubDeviceClient
+from pnp_methods import execute_listener, pnp_update_property, pnp_send_telemetry
+
```

Comment:
When importing a module with functions rather than classes, I think it probably best to just import the module and then access the functions via the module. This makes it clearer what is defined in this file, vs what is imported.

e.g. import pnp_methods
...

pnp_methods.pnp_update_property(...)